libbpf-tools/biolatency.c

```diff
@@ -334,6 +360,7 @@ int main(int argc, char **argv)
 	}
 
 cleanup:
+	close(cgfd);
```

Comment:
Cleanup in reverse order so that BPF program won't reference invalid fd.
Do close only when cgfd > 0.